src/unix/getaddrinfo.c

```diff
@@ -200,3 +200,22 @@ void uv_freeaddrinfo(struct addrinfo* ai) {
   if (ai)
     freeaddrinfo(ai);
 }
+
+
+int uv_if_indextoname(unsigned int ifindex, char* ifname, size_t *sizep) {
+  char ifname_buf[UV_IF_NAMESIZE];
+  if (NULL == if_indextoname(ifindex, ifname_buf)) {
```

Comment:
I don't think it is needed, the underlying implementation should never return anything that is even close to IFNAMSIZ in length.  But I can add such an assignment if you want to have both a belt and suspenders.